src/cmdc_tools/datasets/official/CA/data.py

```diff
@@ -77,5 +102,20 @@ def get_hospital_data(self):
         out = df.melt(
             id_vars=["dt", "county"], var_name="variable_name", value_name="value"
         ).dropna()
+        out["value"] = out["value"].astype(int)
+
+        return out
+
+    def get_test_data(self):
+        resource_id = "b6648a0d-ff0a-4111-b80b-febda2ac9e09"
+        df = self.data_from_api(resource_id=resource_id)
+
+        df = df.rename(columns={"date": "dt", "tested": "tests_total",}).drop(
+            "_id", axis=1
+        )
+        df["dt"] = pd.to_datetime(df["dt"])
+        df["fips"] = self.state_fips
```

Comment:
I don't think this works as a method on the county dataframe.

```
In [4]: df
Out[4]:
            county         dt  fips    value variable_name                   vintage
0      Santa Clara 2020-03-18   NaN      151   cases_total 2020-07-31 00:00:00+00:00
1      Santa Clara 2020-03-19   NaN      183   cases_total 2020-07-31 00:00:00+00:00
2      Santa Clara 2020-03-20   NaN      246   cases_total 2020-07-31 00:00:00+00:00
3      Santa Clara 2020-03-21   NaN      269   cases_total 2020-07-31 00:00:00+00:00
4      Santa Clara 2020-03-22   NaN      284   cases_total 2020-07-31 00:00:00+00:00
...            ...        ...   ...      ...           ...                       ...
61526          NaN 2020-07-25   6.0  7168139   tests_total 2020-07-31 00:00:00+00:00
61527          NaN 2020-07-26   6.0  7296578   tests_total 2020-07-31 00:00:00+00:00
61528          NaN 2020-07-27   6.0  7417866   tests_total 2020-07-31 00:00:00+00:00
61529          NaN 2020-07-28   6.0  7517466   tests_total 2020-07-31 00:00:00+00:00
61530          NaN 2020-07-29   6.0  7633840   tests_total 2020-07-31 00:00:00+00:00
```

Notice it fills the county as NaN

I think this method needs to be in its own class for CA state data

Reply:
Good catch. Updated.